heareval/tasks/pipeline.py

```diff
@@ -282,7 +293,9 @@ def run(self):
             print(f"{num_files} audio files in corpus, keeping only {max_files}")
 
         # Sort by the subsample key and select the max_files number of samples
-        metadata = metadata.sort_values(by="subsample_key").iloc[:max_files]
+        metadata = metadata.sort_values(
+            by=["split_key", "subsample_key"], ascending=[True, True]
+        ).iloc[:max_files]
```

Comment:
Should we issue a warning here if the set doesn't contain examples covering all the labels? I don't think we want to raise an error since it is useful to have small splits for testing and debugging, but if all the labels aren't present then downstream evaluation will break (mostly just AUC).